src/main/java/com/cmsc495/ticketsystem/controller/SubmitTicketController.java

```diff
@@ -66,7 +66,7 @@ public ResponseEntity<String> submitTicket(@RequestParam String name,
 
     private void sendEmail(TicketModel ticketModel, String email, String department, String issueType) {
         try {
-            emailService.sendEmail(email, department + issueType + ticketModel.getFormattedDate(), "Ticket submitted!");
+            emailService.sendEmail(email, department + issueType + ticketModel.getFormattedDate(), "Ticket submitted! A tech will get to work on your ticket immediately. We will keep you updated with the status of your ticket until it is closed. Thank you for your patience.");
```

Comment:
Can you add some spaces here to the subject? As of now all of the text is combined together 


![image](https://github.com/user-attachments/assets/89e9bd52-1792-4014-9490-2b7d53e2e379)

Please attach an image of an example email after